src/sugar3/graphics/toggletoolbutton.py

```diff
@@ -58,6 +84,16 @@ def setup_accelerator(tool_button):
 
 
 class ToggleToolButton(Gtk.ToggleToolButton):
+    '''
+    UI for toggletoolbutton
```

Comment:
Please be more descriptive.  Explain where you put this, what it does, what signals it has (eg. 'toggled')